### PR TITLE
EWPP-4288: Remove already migrated fields.

### PR DESCRIPTION
**Pull request by nagyad**
This must go into 4.x branch!

### CODE REVIEW

File: modules/oe_content_project/README.md [hunk 1]
```diff
@@ -51,16 +51,10 @@ For more information check `cweagans/composer-patches` documentation [here](http
 
 ## Replacing Project budget fields and removing previous float type fields as of 4.x version
 
-As of 4.x the following float type fields will be removed:
+As of 4.x the following float type fields had been removed:
```
Comment (22Alexandra) [inline modules/oe_content_project/README.md:54]: So these fields were supposed to be removed in the next major considering the previous statement. Are we going to 4.x or is it ok to remove the fields in a minor version?

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
Is `AuthorSkosUpdater` still needed?

File: modules/oe_content_project/README.md (hunk 1 above)
Comment (nagyad) [reply modules/oe_content_project/README.md:54]: We will have to merge this into a new 4.x branch and we will go to a new major after this.

**Comment by nagyad** [comment]
The `AuthorSkosUpdater` is needed because we made it as optional since we are not deleting or deprecating oe_author fields in this component, except EWPP where it is a project decision. Therefore, another site/project can decide anytime to move from oe_author to oe_authors sub-entities, which is a good reason to keep the updater service since we are maintaining both fields in this module.

File: modules/oe_content_project/oe_content_project.post_update.php [hunk 2]
```diff
@@ -67,3 +68,18 @@ function oe_content_project_post_update_30001(): void {
...
+ * Remove old budget fields after 4.x.
+ */
+function oe_content_project_post_update_40001(): void {
+  $fields_to_remove = [
+    'oe_project_budget',
+    'oe_project_budget_eu',
+  ];
+  foreach ($fields_to_remove as $field_name) {
+    if ($field_storage = FieldStorageConfig::load("node.$field_name")) {
+      $field_storage->delete();
```
Comment (sergepavle) [inline modules/oe_content_project/oe_content_project.post_update.php:82]: For safety better anyway to check if the field exists before trying to remove it.

**Comment by sergepavle** [comment]
Are we planning still to keep `PersonNodeUpdater` service?

**Review by sergepavle — DISMISSED** [review]

**Review by 22Alexandra — DISMISSED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Pull request merged**